tox.ini

```diff
@@ -44,6 +44,7 @@ deps =
     -e .[doc]
 allowlist_externals = make
 commands =
-    python archive_examples.py
-    make -C doc html
-    make -C doc pdf
\ No newline at end of file
+    sphinx-build -M html "{toxinidir}/doc/source" "{toxinidir}/doc/_build" -j auto
+    sphinx-build -M latex "{toxinidir}/doc/source" "{toxinidir}/doc/_build" -j auto
+    cd "{toxinidir}/doc/_build/latex" && latexmk -r latexmkrc -pdf *.tex -interaction=nonstopmode || true
+    (test -f "{toxinidir}/doc/_build/latex/*.pdf" && echo pdf exists) || exit 1
```

Comment:
This is POSIX syntax. It will not work in Windows. We can remove these lines and only build `html`. This is the usual approach in other libraries.

For the PDF, let's configure that rule in the `Makefile` and `make.bat` files.

Reply:
The rule already exists in Makefile/make.bat.
I will remove the lines and update the repo actions accordingly.